store/src/java/com/zimbra/cs/mailbox/calendar/Invite.java

```diff
@@ -357,6 +416,9 @@ public static Metadata encodeMetadata(Invite inv) {
         }
 
         meta.put(FN_NAME, inv.getName());
+        meta.put(FN_OPERATION_TYPE, inv.getOperationType());
+        meta.put(FN_TAG_COLOUR, inv.getTagColour());
+
```

Comment:
extra space

Reply:
Above line - meta.put(FN_NAME, inv.getName()); has also extra space. Still shall I remove the space. Please confirm.